csharp/ql/src/change-notes/2025-05-02-mad-generator-renamed.md

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
Considering https://github.slack.com/archives/C06016CLLGY/p1747081761540249
then we should probably not have a change note for this PR.

Reply:
Makes sense and done. It's honestly also a bit easier, if we can consider the model generator internal.

Reply:
Yes, I agree - but I think that the field services team uses the model generator queries - so we might break stuff, if we make changes to the queries (however, the script is to my knowledge not used elsewhere)